libsplinter/src/transport/multi.rs

```diff
@@ -31,6 +31,10 @@ impl MultiTransport {
     pub fn new(transports: Vec<SendableTransport>) -> Self {
         Self { transports }
     }
+
```

Comment:
Can't you just pass a clone of the inproc transport before creating the multi transport?

Reply:
We decided to leave this